.buildkite/pipeline.yml

```diff
@@ -1,5 +1,13 @@
 steps:
-  - name: ":golang: test"
-    command: "go test -v ./..."
-  - name: ":golang: vet"
-    command: "go vet ./..."
+  - label: ":go: Run tests"
+    command: |
+      go version
+      go test -v ./...
+    agents:
```

Comment:
maybe put the queue into the steps ui instead. Other than that this looks great!